backend/server.js

```diff
@@ -6,12 +6,26 @@ const morgan = require("morgan");
 const bodyParser = require("body-parser");
 app.use(morgan("tiny"));
 
-const buildurl = (version, path) => `/api/${version}/${path}`;
+const buildurl = path => `/${path}`;
 
-const EVENTS_BASE_URL = buildurl("v1", "events");
+const EVENTS_BASE_URL = buildurl("events");
+
+const FLOATERS_BASE_URL = buildurl("mentors");
 
 app.use(EVENTS_BASE_URL, router);
 
+
+app.use(FLOATERS_BASE_URL, router);
+
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header(
+    "Access-Control-Allow-Headers",
+    "Origin, X-Requested-With, Content-Type, Accept"
+  );
+  next();
+});
```

Comment:
Are these headers still required? I thought having ```proxy``` in ```frontend/package.json``` should do the job.

Reply:
I don't know I'm going to remove them and test it.